### ISSUE TITLE
Feature request: show features actual size instead of scaled to fit the browser

### ISSUE THREAD

**Issue by gammata** [MHeasell/Mappy#17]
Showing features as the size they will actually appear on the map would make it much easier to choose features from the browser. Could be just an option "show features actual size"

**Comment by milkmanmatty** [comment]
When you say browser, are you referring to the list of features in the sidebar?

**Comment by gammata** [comment]
yes, I called it a "browser" because it shows what the features look like. Unfortunately they are all auto-resized to the same size so it's hard to tell what you're getting until you place one on the map
